app/controllers/admin/events_controller.rb

```diff
@@ -8,7 +8,7 @@ def new
   def create
     @event = Event.new(event_params)
     if @event.save
-			@event.event_teammates.create(user: current_user, role: 'organizer')
+			@event.teammates.create(user: current_user, email: current_user.email, role: 'organizer')
```

Comment:
Since we've made teammate create be the invite step then needs to change.  You must manually accept the admin user.  Do a build on teammates passing in email and role, then chain accept passing in the current_user